assets/js/base/components/cart-checkout/address-form/default-address-fields.js

```diff
@@ -70,6 +74,7 @@ const AddressFields = {
 			'woo-gutenberg-products-block'
 		),
 		autocomplete: 'address-line2',
+		autocapitalize: 'words',
```

Comment:
Possibly this could be `sentences`, but I don't think it's a big deal either way. E.g. `Second floor apartment 2`

Reply:
Hmm, now that I've tested, I think we should use `sentences` everywhere :)